.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: circleci/node:8-browsers
+    steps:
+      - checkout
+      - run: yarn install
+      - run: yarn test
+
+  deploy:
+    docker:
+      - image: circleci/python:3.6.5
+    working_directory: ~/circleci-docs-deploy
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "00:00:00:00:00:00:00:00:00:00:00:00:00:00:00:00"
```

Comment:
This line needs to be updated to the real fingerprint.

Reply:
Is this necessary given that CircleCI is already authenticated with my account for this project?  mkdocs deploy was working before without this.

Reply:
You want tests to run on PRs with a read-only user, ideally, to limit attack surface. This config line allows for granting write access to only the deploy job, which only runs when a commit is marked with a version tag.

Yes, everything will still work. This just bolsters security.

Reply:
Sorry, I'm a bit naive on this.  I believe mkdocs works by pushing back the gh-docs branch, so that would not be read-only.  Have you used mkdocs on a different repo with a similar circle config?  Any examples I can take a look at so that I can finish setting this up?

Reply:
Can we just go on and merge the CI 2.0 code into master? What could go wrong besides another failing CircleCI 2.0 and some shame that you might have overlooked something? In my understanding we currently can only test the config when another tag is created.

Ok. Tagging another version just for trying out your configuration might not be the best idea.

Or should we setup another CircleCI test branch for testing the deployment of generated documentation? I can try it out on my branch, where I had some problems with pushing the latest docs. I'll see if I can do this tonight.

Reply:
Agreed